bin/package-requires-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var packageJson = require( '../' + process.argv[2] );
+var semver = require( 'semver' );
+
+if ( packageJson.private ) {
+	process.exitCode = 0;
```

Comment:
0 is the default exit code, so isn't this basically meaningless?  Did you mean `process.exit(0)`?